### PR TITLE
Hid unimportant hoaxy ui elements on load

### PR DESCRIPTION
**Pull request by MaxReinisch**

### CODE REVIEW

File: scripts/contentScript.js [hunk 1]
```diff
@@ -1,9 +1,15 @@
 window.onload=function(){
-    console.log("Hi");
-    setTimeout(
-        function() {
-            console.log("here");
-            document.querySelector('.layout-button-graph').click();
-        },
+  console.log("Hi");
```
Comment (vbanos) [inline scripts/contentScript.js]: Do we need this in the final code?

**Comment by vbanos** [comment]
I don't see any other issue with your PR. I'd be happy to merge after you fix that minor thing I mentioned. Thank you!

**Comment by MaxReinisch** [comment]
Would you mind explaining further? I didn't understand what exactly you
meant by "this".

**Comment by anishsarangi** [comment]
@MaxReinisch Please remove all the console.log statements in your code. We don't need this in our final code.
Thank you !!

**Pull request merged**